xio-core/src/main/java/com/xjeffrose/xio/http/RouteState.java

```diff
@@ -25,4 +26,17 @@ public RouteState(
   public RouteState(RouteConfig config, PipelineRequestHandler handler) {
     this(RouteState::buildRoute, config, handler);
   }
+
+  public static RouteState defaultRoute(PipelineRequestHandler handler) {
+    RouteConfig config = new RouteConfig(ConfigFactory.load().getConfig("xio.defaultRoute"));
```

Comment:
Should all these string literal put into constants somewhere? e.g., "xio.defaultRoute" and "*" on the next line.

Reply:
my rule of thumb with constants is to create one whenever I'm about to copy/paste a value.